### DISCUSSION TITLE
Explain to me

### DISCUSSION THREAD

**Discussion by AAKboss** [TodePond/GulfOfMexico#313] [Q&A] [answered]
So, if "When" is python's "if", wouldn't that allow for loops?
But then it was stated that C removed loops?
So, is this a glitch in the system or something I am failing to understand??

**Comment by TodePond** [comment, marked as answer]
hi technically it's a "repetition", not a "loop", so it's ok

**Comment by stohrendorf** [comment]
If I understand it correctly, `when` is a conditional event handler, comparable to a `whif`.

**Reply by AAKboss** [reply]
What's a `WHIF` ??

**Reply by stohrendorf** [reply]
A "when-if".
